AdyenDropIn/Presentation/WrapperViewController.swift

```diff
@@ -52,36 +48,51 @@ internal final class WrapperViewController: UIViewController {
                                                                self?.update(finalFrame: finalFrame)
                                                            }))
     }
-    
-    private func update(finalFrame: CGRect) {
-        guard let view = child.viewIfLoaded else { return }
-        leftConstraint?.constant = finalFrame.origin.x
-        rightConstraint?.constant = -finalFrame.origin.x
-        topConstraint?.constant = finalFrame.origin.y
-        view.layoutIfNeeded()
-    }
 
-    fileprivate func positionContent(_ child: ModalViewController) {
+    // MARK: - Private
+
+    private func setupChildViewController() {
         addChild(child)
-        view.addSubview(child.view)
         child.didMove(toParent: self)
```

Comment:
this has to be after `view.addSubview(child.view)`, like this:

```
addChild(child)
view.addSubview(child.view)
child.didMove(toParent: self)
setupChildLayout()
```

this is more accurate

Reply:
I agree, done